demos/vector_graphic/lv_demo_vector_graphic.c

```diff
@@ -228,20 +243,32 @@ static void draw_vector(lv_layer_t * layer)
 
     draw_shapes(ctx, path);
     draw_lines(ctx, path);
-    draw_pattern(ctx, path);
+    if(is_partially_buffered == 0) {
+        /*TODO: Currently not supported in partial mode*/
```

Comment:
Please replace the TODO with a relevant LOG call (a LOG_WARN should be sufficient)